_posts/2018-07-25-dota-macos-performance.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: Portability benchmark of Dota2 on MacOS
+---
+
+### The Race
+
+gfx-rs is a Rust project aiming to make graphics programming more accessible and portable, focusing on Vulkan. It's a single Rust API with multiple backends: D3D12/11, Metal, Vulkan, and even OpenGL. We are also building a Vulkan Portability [implementation](https://github.com/gfx-rs/portability) based on it, which allows non-Rust applications using Vulkan to run everywhere.
```

Comment:
Maybe don’t abbreviate Direct3D here (for people unfamiliar with the abbreviation)